css/danti.html

```diff
@@ -0,0 +1,127 @@
+                </div>
+                <div class="first_last">
+                    <div class="first_name">
+                        <label for="name">First name *</label><br>
+                        <input type="text" id="name_input" name="name" value=""/>
+                    </div>
+        
+                    <div class="last_name">
+                        <label for="last_name">Last name *</label><br>
+                        <input type="text" id="last_name_input" name="last_name" value=""/>
+                    </div>
+                </div>
+                
+                <div class="address">
+                    <label for="adress">Address *</label><br>
+                    <input type="text" id="addres_input" name="address" value=""/>
+                </div>
+    
+                <div class="address_2">
+                    <label for="address_2">Address 2</label><br>
+                    <input type="text" id="address_input_2" name="address_2" value="">
+                </div>
+                <div class="city_postcode">
+                    <div class="city_city">
+                        <label for="city">City *</label><br>
+                        <select id="city" name="city">
+                            <option class="city_input" value="Select" selected aria-placeholder="Select your city..."></option>
+                            <option value="Eri">Eritria</option>
+                        </select>
+                    </div>
+                    <div class="postcode">
+                        <label for="city">Post code *</label><br>
+                        <input type="text" id="postcode_input" name="city" value=""/>
+                    </div><br>
+                    </div>
+                    <div>
+                        <label for="color" class="color">Select a colour</label><br>
+                        
+                        <div class="color_radio">
+                            <div class="karma_orange">    
+                                <input id="orange" class="orange" type="radio" name="size" value="Karma orange" />
+                                <div class="lab_1"><label for="small">Karma orange</label></div>
+                            </div>
+                            <div class="karma_grey">   
+                                <input id="grey" type="radio" name="size" value="Space Grey" checked />
+                                <div class="lab_2"><label for="grey">Space Grey</label></div>
+                            </div>     
+                        </div>
+                    </div>
+                    <div class="checkbox_1">
+                        <div class="term">   
+                            <input id="term_input" type="checkbox" name="size" value="Terms" checked />
+                            <div class="term_text">
+                            <label for="grey">By  placing your order you agree to karma's <span>Terms and <br>Conditions</span></label>
```

Comment:
Again, just need to match the value of the `for` attribute to the input :slightly_smiling_face: 
```suggestion
                            <label for="term_input">By  placing your order you agree to karma's <span>Terms and <br>Conditions</span></label>
```